test/testConstants/ChargeConstants.scala

```diff
@@ -544,6 +543,8 @@ trait ChargeConstants {
     transactionTypes = List(PoaOneReconciliationDebit, PoaTwoDebit),
     dueDate = List(Some(fixedDate.plusYears(100).minusDays(1)), Some(fixedDate.plusYears(100).plusDays(30))),
     outstandingAmount = List(50, 75),
+    outstandingInterest = List(Some(BigDecimal(10)),Some(BigDecimal(20))),
```

Comment:
it is optional but generally if we can use amount with pence included to cover more scenarios